src/core/QCheck.ml

```diff
@@ -1506,10 +1543,11 @@ module Test = struct
     let hist_size, bucket_size =
       let sample_width = Int64.(sub (of_int max_idx) (of_int min_idx)) in
       if sample_width > Int64.of_int stat_max_lines
-      then 1+stat_max_lines,
```

Comment:
does this belong in the other PR?

Reply:
Hm. That's weird. Indeed it has nothing to do with list shrinking (and was committed a day earlier). I *really* should start using branches... :-)